components/core/src/os/users/mod.rs

```diff
@@ -16,10 +16,10 @@
 mod windows;
 
 #[cfg(windows)]
-pub use self::windows::{get_uid_by_name, get_gid_by_name, get_effective_uid, get_home_for_user};
+pub use self::windows::{get_uid_by_name, get_gid_by_name, get_effective_uid, get_home_for_user, get_current_username, get_current_groupname};
```

Comment:
We can use a `*` to export all the functions, structs, etc in a module.  In this case, we'd want to bury `GetUserTokenStatus` because a wildcard would export that unsafe function which is really more of an internal implementation detail.  (I think we can just make that function not public and we should be alright.) 

Rust style/patterns tend to lean toward explicit imports and exports so you know what you are getting from where.

There's a good thread on the Rust forums on this topic as well.. 

https://users.rust-lang.org/t/confused-on-mod-use-pub-simple-example/1973/6?u=smurawski